tests/callbacks/test_async_eval_callback.py

```diff
@@ -0,0 +1,237 @@
+# Copyright 2022 MosaicML LLM Foundry authors
+# SPDX-License-Identifier: Apache-2.0
+
+import datetime
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from llmfoundry.callbacks.async_eval_callback import AsyncEval, get_run_name
+from mcli import Run, RunConfig, RunStatus
+
+RUN_NAME = 'foo_bar-1234'
+
+
+def test_get_run_name():
+    a = get_run_name('foo-1234', 0)
+    assert a == 'eval0-foo'
+
+    b = get_run_name(50 * 'foo' + '-1234', 1)
```

Comment:
what is this testing? something about name length?

Reply:
If the name is too long run creation will fail. Can add a small comment